src/store/profile.js

```diff
@@ -137,8 +145,31 @@ export const useProfileStore = defineStore({
 
       const profile = await this.getProfile(id)
       this.setActiveProfile(profile)
+      await this.loadProfileThreads()
 
       return profile
+    },
+    /**
+     * Fetches the threads from user with a particular feedId
+     */
+    async loadProfileThreads () {
+      const res = await apolloClient.query({ query: GET_PROFILE_THREADS, variables: { id: this.activeProfile.id, cursor: null } })
```

Comment:
I think you want a `limit` here right? I think the API has a default but putting it explicitly helps to understand how this works and is a hint for customisation to future devs